src/gpb_compile.erl

```diff
@@ -1820,6 +1820,9 @@ format_erl(Mod, Defs, #anres{maps_as_msgs=MapsAsMsgs}=AnRes, Opts) ->
        %% to about 10000 msgs/s for a set of mixed message samples.
        %% f("-compile(inline).~n"),
        %%
+       %% Add epb compatibility function
+       ?f("-spec encode(_) -> no_return().~n"),
+       ?f("encode(Msg) -> encode_msg(Msg, []).~n"),
```

Comment:
Yes, I thought this wasn't correct either. I got the idea for this spec from here:

https://github.com/tomas-abrahamsson/gpb/blob/master/src/gpb_compile.erl#L1913

I'll see about changing it...